examples/megacli.cpp

```diff
@@ -3907,6 +3942,26 @@ void exec_sync(autocomplete::ACState& s)
         }
     }
 }
+
+void exec_syncconfig(autocomplete::ACState& s)
+{
+    if (s.words.size() == 1)
+    {
+        cout << "Current sync config: " << syncConfig.syncType <<
+                " " << syncConfig.syncDeletions << " " << syncConfig.forceOverwrite << endl;
+    }
+    else if (s.words.size() == 4)
+    {
+        syncConfig.syncType = static_cast<SyncConfig::Type>(atoi(s.words[1].s.c_str()));
```

Comment:
We could make it more intuitive by avoiding the user to know the enumeration values. In example, instead of `0` for up-sync, `UP` (`DOWN` and `FULL`). Fine with 0 or 1 for the flags ;)

Reply:
Thank you. It is a dev tool, indeed, but that's a small cost that I feel will be paid back after a few months, when another dev than the author wants to try or needs to debug this new command.